dashboard/test/models/pd/scholarship_info_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class Pd::ScholarshipInfoTest < ActiveSupport::TestCase
+  include Pd::ScholarshipInfoConstants
+  include Pd::Application::ActiveApplicationModels
+
+  self.use_transactional_test_case = true
+
+  setup_all do
+    @user = create :teacher
+  end
+
+  test 'application year is set on create' do
+    scholarship_info = Pd::ScholarshipInfo.create(user: @user, scholarship_status: YES_CDO)
+
+    assert_equal(APPLICATION_CURRENT_YEAR, scholarship_info.application_year)
+  end
+
+  test 'cannot create multiple scholarship infos with same user and year' do
+    Pd::ScholarshipInfo.create(user: @user, scholarship_status: YES_CDO)
+
+    assert_raises(ActiveRecord::RecordNotUnique) do
+      Pd::ScholarshipInfo.create(user: @user, scholarship_status: YES_CDO)
+    end
+  end
+
+  test 'application year must be in SCHOLARSHIP_YEARS' do
```

Comment:
I'm feeling like these 2 tests `application year must be in SCHOLARSHIP_YEARS` and `scholarship status must be in SCHOLARSHIP_STATUSES` are to make sure `validates_inclusion_of` works as intended, which is almost guaranteed. Are they overkill?

Reply:
I see your point, and I am happy to take them out if they seem unnecessary. I don't feel strongly either way.

Reply:
I think they could be removed. We are also having a bigger discussion on how we should test driven by ErinP. To me the result of testing is the **confidence** that our program do the right things. Some confidence is already provided by the platform/lib we use so we don't have to re-test it (unless there is doubt).